common/djangoapps/student/emails.py

```diff
@@ -0,0 +1,34 @@
+"""
+ACE emails for Student app
+"""
+import logging
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+from edx_ace import ace
+from edx_ace.recipient import Recipient
+
+from common.djangoapps.student.message_types import ProctoringRequirements
+from openedx.core.djangoapps.ace_common.template_context import get_base_template_context
+
+log = logging.getLogger(__name__)
+
+
+def send_proctoring_requirements_email(context):
```

Comment:
I saw in your test_emails unit tests, you used the enrollment to trigger this function.
I think you should organize the unit tests better to unit test this particular function directly, then trigger the enroll function with other enroll related logic.
Put it a different way, I think you should add unit tests on this particular new function, even if the case isn't many.